cms/static/js/views/modals/base_modal.js

```diff
@@ -34,6 +34,7 @@ define(["jquery", "underscore", "gettext", "js/views/baseview"],
                 modalType: 'generic',
                 modalSize: 'lg',
                 title: '',
+                modalWindowClass: '.modal-window',
```

Comment:
Please replace the other places in this file that specify ".modal-window" with your new variable.